src/inference/src/system_conf.cpp

```diff
@@ -225,23 +230,27 @@ std::vector<int> get_available_numa_nodes() {
 #    endif
 
 std::vector<std::vector<int>> get_proc_type_table() {
+    CPU& cpu = cpu_info();
     std::lock_guard<std::mutex> lock{cpu._cpu_mutex};
     return cpu._proc_type_table;
 }
 
 bool is_cpu_map_available() {
+    CPU& cpu = cpu_info();
+    std::lock_guard<std::mutex> lock{cpu._cpu_mutex};
```

Comment:
Why we need additional lock when we only check if cpu map is exist?

Reply:
We can remove the lock when "numactl" command is supported. Because we will  access _cpu_mapping_table instead of _proc_type_table.